index.html

```diff
@@ -7412,8 +7893,9 @@ <h2>Conformance of PNG
 <!-- Maintain a fragment named "IANA-registrations" to preserve incoming links to it -->
 <h2 id="IANA-registrations">Internet Media Types</h2>
 
-<!-- Maintain a fragment named "image-PNG-EXTENSIONSistration" to preserve incoming links to it -->
-<h2 id="image-PNG-EXTENSIONSistration">image/png</h2>
+<!-- Maintain a fragment named "A-Media-type" to preserve incoming links to image/png registration -->
+<section>
+<h3 id="A-Media-type">image/png</h3>
```

Comment:
We can lose the id & comment

Reply:
I disagree, because [PNG second edition has it](https://www.w3.org/TR/PNG/#A-Media-type) `https://www.w3.org/TR/PNG/#A-Media-type` so I was restoring a link that seems to have been changed in an earlier edit.